docs/src/development/ssh.md

```diff
@@ -2,131 +2,118 @@
+...(many more lines like this)...
 QGerp3VKaGe0St3ot57GlwCAQUJAf1mit8qDTi0I8MhBe7q2lstXkBvde7GY1gKx
 Kng4ohG6xHZ/OvC9tq7/THwAvleaxgLZN5GyXfAqNylDdZ0LtSjl
 -----END RSA PRIVATE KEY-----
 ```
 
-*A public key (one very long line)*:
+A public key (one very long line):
 
 ```text
 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC2nDRLgPANWParTiaGIgySG+thTtnqFGI1tMWyqDdfvH+5hL91w2tK9PzaP+NJ5hA/cOyh30YRFb52Y64toU16Ko5K1mLqNFJajjWEI5Y4VukG6betrWfqdQ7XBr/s7nBuDOFQ5+eKbvug4rRSCSo8CsEI1eI0VNQkC9HJWYK28k7KurMdTN7X/Z/4vknM4/Rm2bnMk2idoORQgomeZS1p3GkG8dQs/c0j/b4H7azxnqdcCaR4ahbytX3d49BN0WwE84C+ItsnkCt1g5tVADPrab+Ywsm/FTnGY3cJKKdOAHt7Ls5lfpyyug2hNAFeiZF0MoCekjDZ2GH2xdFc7AX/ your_email_address@example.com
 ```
 
-GitHub has a good [walk-through of creating an SSH keypair](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) on various operating systems.
+A key pair is valid for as long as you have access to the private key on the system from which you are connecting. If you have a key pair available, you are not prompted to login.
 
-A keypair is valid for as long as you have access to the private key on the system from which you are connecting.  If you have a keypair available you will not be prompted to login.
+If you used GitHub to sign up for your Platform.sh account, your public keys from GitHub are automatically synced to your Platform.sh account. So you can use them already with the CLI or to [connect to a server](#connect-to-your-server-with-SSH).
 
-### Find your Public-Private keypair
+Otherwise, you might be able to [find existing keys](#find-your-keys) or else you need to [generate new keys](#generate-new-keys).
 
-If you use Linux, you probably already have keys. The private key is usually in a file named `~/.ssh/id_rsa` and the public key in `~/.ssh/id_rsa.pub`.
+### Find your keys
 
-Searching for a public key file:
-1. Open up a command prompt.
-2. Run the following commands:
+If you use Linux, you probably already have keys. The private key is usually in a file named `~/.ssh/id_rsa` and the public key in `~/.ssh/id_rsa.pub`. Tech Republic has a guide to [finding keys on different systems](https://www.techrepublic.com/article/how-to-view-your-ssh-keys-in-linux-macos-and-windows/).
 
-    ```bash
-    $ cd ~/.ssh
-    $ ls -a
-    id_rsa
-    id_rsa.pub
-    known_hosts
-    authorized_keys
-    ```
+To find your public key file:
 
-    If you find a file named `id_rsa.pub`, you can use it with Platform.sh. If you don't find an existing key, see the steps to create a new one in the [next section](#create-a-new-public-private-keypair).
+1. Open a terminal.
+2. Run the following commands:
 
-### Create a New Public-Private Keypair
+   ```bash
+   $ cd ~/.ssh
+   $ ls -a
+   ```
 
-{{< note >}}
-If you already have a SSH keypair, you can skip this step.
-{{< /note >}}
+If you find a file ending in `.pub`, copy the location and [add it to your Platform.sh account](#add-an-ssh-key-to-your-platform-account).
```

Comment:
shall we display the command to copy it automatically or is it too complex here?

` pbcopy < ~/.ssh/name_of_the_key.pub`

Reply:
I think `pbcopy` is exclusive to macOS. I could add in tabs with commands by OS, but that might be too complicated. What do you think?